client/devdocs/main.jsx

```diff
@@ -1,28 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	isFunction = require( 'lodash/isFunction' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import isFunction from 'lodash/isFunction';
 
 /**
  * Internal dependencies
  */
-var DocService = require( './service' ),
-	Card = require( 'components/card' ),
-	Main = require( 'components/main' ),
-	SearchCard = require( 'components/search-card' );
+import DocService from './service';
+import Card from 'components/card';
+import Main from 'components/main';
+import SearchCard from 'components/search-card';
+import { decodeEntities } from 'lib/formatting';
```

Comment:
In server/devdocs/index.js, in defaultSnippet(), the `&hellip;` is added after generating the snippet for the required length. So there will be no apparent advantage. 

Additionally, it was my understanding that the server side should return HTML that has been escaped in order to prevent XSS attacks.

Lastly, I think it would be good practice to decode the html entities closest to where it is going to be displayed, wdyt?